src/main/java/sggw/wzim/czasnawypad/controller/RestExceptionHandler.java

```diff
@@ -55,8 +56,32 @@ public ResponseEntity<ErrorResponseDTO> handleUsernameNotFoundException(Exceptio
         return handleExceptionInternal(status);
     }
 
+    @ExceptionHandler(ApplicationExceptions.AttractionNotFoundException.class)
+    public ResponseEntity<ErrorResponseDTO> handleAttractionNotFoundException(ApplicationExceptions.AttractionNotFoundException ex) {
+        log.error(ex.getMessage(), ex);
+        HttpStatus status = HttpStatus.NOT_FOUND;
+        return handleExceptionInternal(status, ex.getMessage());
+    }
+
+    @ExceptionHandler(ApplicationExceptions.UserNotFoundException.class)
+    public ResponseEntity<ErrorResponseDTO> handleUserNotFoundException(ApplicationExceptions.UserNotFoundException ex) {
+        log.error(ex.getMessage(), ex);
+        HttpStatus status = HttpStatus.NOT_FOUND;
+        return handleExceptionInternal(status, ex.getMessage());
+    }
+
+    @ExceptionHandler(ApplicationExceptions.RatingNotFoundException.class)
+    public ResponseEntity<ErrorResponseDTO> handleRatingNotFoundException(ApplicationExceptions.RatingNotFoundException ex) {
+        log.error(ex.getMessage(), ex);
+        HttpStatus status = HttpStatus.NOT_FOUND;
+        return handleExceptionInternal(status, ex.getMessage());
+    }
```

Comment:
Since these are using the same `HttpStatus.NOT_FOUND`, please group them into one handler